src/main/java/org/isf/visits/model/Visit.java

```diff
@@ -39,6 +39,8 @@
 import javax.persistence.Transient;
 import javax.validation.constraints.NotNull;
 
+import org.apache.log4j.AppenderSkeleton;
```

Comment:
What is this import used for?

Reply:
really don't know why was there, fixed. Thanks